xtask/src/main.rs

```diff
@@ -0,0 +1,69 @@
+use std::{env, process::Command};
```

Comment:
I would appreciate a small (3-5 lines?) docstring here which explains what this binary is for.

Reply:
Feel free to do this in a later PR!

Reply:
Have pushed a new PR (https://github.com/google/comprehensive-rust/pull/2707) for this and the other suggestions.